horace_core/sqw/@sqw/rebin_sqw.m

```diff
@@ -22,118 +22,118 @@
 %
 % RAE 21/1/10
 
-    if ~isa(win, 'sqw')
-        %% TODO what we should actually do here is go to the dnd-rebin of correct
-        % dimensionality
-        error('HORACE:rebin_sqw:invalid_argument', ...
-              'input object must be sqw type with detector pixel information');
-    end
-
-    if isempty(varargin)
-        error('HORACE:rebin_sqw:invalid_argument', ...
-              'requires at least one other argument');
-    end
+if ~isa(win, 'sqw')
```

Comment:
It's not possible for `win` to not be an `sqw` here as I understand it.

Reply:
Correct statement but to be fair, I have not touched `rebin_sqw` -- it should be different ticket. I had a look at it dealing with slim and modified spaces as it happens almost automatically. `rebin` should be overlooked and modified properly but it is technical dept undelated to `slim`.
If you do comparison ignoring changes in spaces you will not see changes to rebin